clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/view/provider/KubernetesV2LiveManifestProvider.java

```diff
@@ -84,7 +90,10 @@ public KubernetesV2Manifest getManifest(String account, String location, String
     String namespace = manifest.getNamespace();
     KubernetesKind kind = manifest.getKind();
 
-    List<KubernetesManifest> events = credentials.eventsFor(kind, namespace, parsedName.getRight());
+    List<KubernetesManifest> events =
+        !includeEvents
```

Comment:
This would be easier to read if we checked the positive condition and inverted to:
```
includeEvents ? credentials.eventsFor(kind, namespace, parsedName.getRight()) : Collections.emptyList()
```